azurerm/internal/services/network/subnet_resource.go

```diff
@@ -69,6 +69,7 @@ func resourceSubnet() *schema.Resource {
 				Type:     schema.TypeList,
 				Optional: true,
 				Computed: true,
+				MinItems: 1,
```

Comment:
alternatively we could make this and the `address_prefix` field uncomputed, set `address_prefix` to an empty value and then push people across - ignoring the value for this older field, since this has been superseded for a while now?

Reply:
Perhaps we'd better do this when bumping towards 3.0 as it still might break some existing configs?

Reply:
its only been superseded for a couple of releases i think